src/tests/suite/daos_test_common.c

```diff
@@ -1072,25 +1073,29 @@ get_server_config(char *host, char *server_config_file)
 			break;
 	}
 
-	pch = strtok(line, " ");
-	while (pch != NULL) {
-		if (strstr(pch, "--config") != NULL) {
-			if (strchr(pch, '=') != NULL)
-				strcpy(server_config_file,
-				       strchr(pch, '=') + 1);
-			else {
+	if (conf)
```

Comment:
I don't quite follow this: The 'conf' is initialized as 'true', so what's the purpose of else branch?

Reply:
Well, there are multiple ways to start the daos serves. in Ci, we moved to start as systems with default config path. But if someone tried to run locally without systemd, the test also needs to find the correct config so that is the reason for the else part.